sycl/include/CL/sycl/context.hpp

```diff
@@ -217,12 +217,10 @@ class __SYCL_EXPORT context {
   /// Gets the native handle of the SYCL context.
   ///
   /// \return a native handle, the type of which defined by the backend.
-  template <backend BackendName>
+  template <backend Backend>
   __SYCL_DEPRECATED("Use SYCL 2020 sycl::get_native free function")
-  auto get_native() const ->
-      typename detail::interop<BackendName, context>::type {
-    return reinterpret_cast<
-        typename detail::interop<BackendName, context>::type>(getNative());
+  auto get_native() const -> backend_return_t<Backend, context> {
```

Comment:
```suggestion
  backend_return_t<Backend, context> get_native() const {
```
I found trailing return type less readable. There is no reason to use it in this case and SYCL spec also doesn't use it.

Reply:
@bader Thank you for the suggestion. I've applied the change for this header as well as the others that define the `get_native()` method.